bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/conditions/evaluation/BasePermissionEmittingService.java

```diff
@@ -149,15 +172,15 @@ private ServiceInvocationResult<Emission> fail(ConditionPack pack, TroubleCode c
 	}
 
 	private ZonedDateTime from(ValidityPeriod period) {
-		if (ValidityPeriodClosed.class.isInstance(period)) {
+		if (period instanceof ValidityPeriodClosed) {
```

Comment:
What is the difference that makes you do the alternation?

Reply:
From my experience instanceof is the usual way to check for a type in cases like this and for example Sonar-Lint says the same.
But if you prefer to keep it as it is, I can revert it.

Reply:
It's better for MR to stay focused on the task and not to contain irrelevant changes. Please revert.